CSharpOOP3/CSharpOOP3/Ball.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+namespace CSharpOOP3
+{
+    public class Ball : Shape
+    {
+        #region Fields
+
+        public string Name;
```

Comment:
there is no need to create a public field name in each class, since we have it in common for everyone and was created in the parent class. You can make it public in the parent class or use a public property to get the value from a private field.

Reply:
Fixed